docs/spfx/viva/design/designing-quick-view.md

```diff
@@ -93,7 +93,7 @@ Currently dark themes are not supported for quick views on iOS and Android mobil
 
 :::image type="content" source="../../../images/viva-design/img_quickview_ios_theme.jpg" alt-text="Screenshot that shows how a card viewed on mobile appears in light mode and dark mode.":::
 
-We plan to introduce dark theme support in Quick View in Viva Connections Mobile for both iOS and Android in early Q4 of CY23. The API ([context.hostContext.theme](/javascript/api/sp-adaptive-card-extension-base/ihostcontext)) to get the theme in which Quick View is rendered of the Quick View will be available in SPFx 1.18.0 package.
+We plan to introduce dark theme support in Quick View in Viva Connections Mobile for both iOS and Android in early Q4 of CY23. ([context.sdks.microsoftTeams.teamsJs.app.getContext()](/javascript/api/@microsoft/teams-js/app.context)) to get the theme in which Quick View is rendered of the Quick View will be available in SPFx 1.18.0 package.
```

Comment:
I think we can get a link for our PartialSdk here, 

It may look like /javascript/api/sp-adaptive-card-extension-base/iPartialSDKs kind of
we might get it https://learn.microsoft.com/en-us/javascript/api/sp-adaptive-card-extension-base/adaptivecardextensioncontext?view=sp-typescript-latest here, 

@AJIXuMuK can you help us get this link, when will this be available, or am i missing something?

Reply:
The link will be available after we publish the first version of 1.18.1. Will share the link later.

Reply:
@sharshach - https://learn.microsoft.com/javascript/api/sp-adaptive-card-extension-base/ipartialsdks?view=sp-typescript-latest